[DOCS] Adds GET, GET stats and DELETE inference APIs (#50224)
elastic/elasticsearch: docs/reference/ml/ml-shared.asciidoc

expression.
end::datafeed-id-wildcard[]

tag::decompress-definition[]

> Specifies if the included model definition should be returned as a JSON map or 

Reviewer: I found it took me a re-read to figure out what "true" mapped to, so it might be helpful to be more explicit:
```suggestion
Specifies whether the included model definition should be returned as a JSON map (`true`) or 
```

> in a custom compressed format. Defaults to `true`.

Reviewer: If suggestion on previous sentence is accepted, this one should be too:
```suggestion
in a custom compressed format (`false`). Defaults to `true`.
```

A list of job groups. A job can belong to no groups or many.
end::groups[]

tag::include-model-definition[]
Specifies if the model definition should be returned in the response. Defaults 
to `false`. When `true`, only a single model must match the ID patterns 

> provided, otherwise a bad request is returned.

Reviewer: > otherwise a bad request is returned.

It seems like this is referring to a 400 response code, so perhaps this circumstance should be added to the "response codes" section.

--
end::mode[]

tag::model-id[]

> The unique identifier of the trained {dfanalytics} model.

Author: ```suggestion
The unique identifier of the trained {infer} model.
```
